monorepo/website/src/pages/news/2024-11-01-autumn-update.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Pathoplexus Autumn Update"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
+</div>
+
+# Two Weeks of Pathoplexus
+
+*By the Pathoplexus Team - 1 November 2024*
+
+Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community. In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!
+
+As the seasons turn, we wanted to share some of our recent news:
+
+### In the media:
+- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)
+
+### Visitors and data:
+- Since launch, Pathoplexus had ingested XX new sequences from INSDC-member databases since launch
+- We are excited to have hit 100 users! (Necessary to upload data but not to view data)
+
+### Executive Board update:
+- See recent minutes of [Pathopexus Members meetings, Executive Board meetings, and Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)
+
+### Scientific Advisory Board:
+- We’re incredibly pleased to welcome Meera Chand from the UKHSA to the [Scientific Advisory Board](https://pathoplexus.org/about/sab)!
+
+### Pathoplexus Association:
+- Pathoplexus is now registered as a tax-exempt organization in Switzerland - any donations made by Swiss residents are tax-deductible
+
+### Software updates:
+- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages
+- It's now possible to download both the sequence and metadata for individual sequences easily from their 'detail view' page
+- Group names and information is now easily editable
+- Multi-sequence downloads now have detailed file names, featuring the organism, date, and data type (amino-acid, aligned nucleotide, etc)
+- We've improved how we store author names so that we can more flexibly display them, and pass them in preferred formats on when depositing to INSDC
```

Comment:
Maybe overkill, but VS Code has little screencasts showing new features in action, it helps discoverability of the features, makes it easy to know how to use them. We don't need to add them to all issues, only to those where we think this is nice

Reply:
Sorry I think I don't understand. Would we then add links to these screencasts from the text? or can we put them in the news somehow? I mean that sounds fun if it's not hard, but not fully understanding the suggestion I think

Reply:
We should be able to add them simply like images (as long as they are gifs, so very low tech, just `!
```suggestion
### Software updates:
- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages
  ![/public/news_oct/feature1.gif]
...
```

Reply:
If you're open to this, I might have a look at adding one as a trial, but don't block on this

Reply:
I think I've managed to add one gif now